### ISSUE TITLE
Front end build fails on routes.tsx

### ISSUE THREAD

**Issue by donlyon** [vaadin/hilla#2715]
### Describe the bug

I'm getting the following error during the front end build in production mode.  It does not happen every time, but I first started getting it after updating from 24.5.0.alpha10 to 24.5.0.alpha12.   I reverted but I'm still seeing it now, even after doing full cleans, and even fresh checkouts.

[ERROR] vite v5.4.0 building for production...
[ERROR] transforming...
[ERROR] src/main/frontend/generated/routes.tsx(37,24): error TS2307: Cannot find module 'Frontend/generated/file-routes' or its corresponding type declarations.
[ERROR] ✓ 1801 modules transformed.
[ERROR] -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.vaadin:vaadin-maven-plugin:24.5.0.alpha10:build-frontend (frontend) on project bifrost: Execution frontend of goal com.vaadin:vaadin-maven-plugin:24.5.0.alpha10:build-frontend failed: Vite process exited with non-zero exit code.
Stderr: 'vite v5.4.0 building for production...
transforming...
src/main/frontend/generated/routes.tsx(37,24): error TS2307: Cannot find module 'Frontend/generated/file-routes' or its corresponding type declarations.
✓ 1801 modules transformed.

### Expected-behavior

Successful build

### Reproduction

It happens during production builds, but not consistently

### System Info

MacOS
Vaadin 24.5.0.alpha10/13
Spring boot 3.3.3

**Comment by cromoteca** [comment]
Hi, thank you for reporting.

I cannot replicate the issue. I tried upgrading to alpha 12 and 13, I see some errors related to peer dependencies. Those are known and fixed in https://github.com/vaadin/hilla/pull/2699. I had to remove `node_modules` and `package-lock.json` to test the production build, but otherwise it works and I cannot see the error you mention.

There must be something specific to your project. If you are able to provide more details, that can help identifying the real issue.
